tele_giphy/game/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
-# Create your models here.
+# Keeps tabs on a game token, whether game has started, and ended
+class GameToken(models.Model):
+    game_token = models.CharField(max_length=16)
+    game_active = models.BooleanField(default='False')
+    game_over = models.BooleanField(default='False')
+
+# Can be used for perma users
+class UserToken(models.Model):
+    user_token = models.CharField(max_length=16)
+
+# Keeps track of users in relation to game tokens
+class User(models.Model):
+    username = models.CharField(max_length=60)
+    user_token = models.ForeignKey(UserToken, on_delete=models.CASCADE)
```

Comment:
Maybe we can build it one to one. But if we were to assume there are persistent users, then we'll need one user to multiple tokens.